summernote/summernote
# Support keydown event to record undo history

Improved keydown event  to record  undo history 


#### What does this PR do?

- add undo history when keydown is fired
- exclude keyMap action 

#### Where should the reviewer start?

- start on the src/js/base/module/Editor.js

#### How should this be manually tested?

- npm run dev 
- open summernote 
- press key continuesly.
- make sure your character is erased one by one.

#### Any background context you want to provide?


#### What are the relevant tickets?

#3491

#### Screenshot (if for frontend)


### Checklist
